scripts/test_py3.sh

```diff
@@ -3,7 +3,7 @@
 pytest . \
        --ignore=openlibrary/catalog/marc/tests/test_get_subjects.py \
        --ignore=openlibrary/catalog/marc/tests/test_parse.py \
-       --ignore=openlibrary/tests/catalog/test_get_ia.py \
+       --ignore=openlibrary/catalog/add_book/tests/test_add_book.py \
```

Comment:
@cclauss I'm not sure why the tests started failing recently -- this test is supposed to be ignored as part of this PR -- `test_get_ia` now passes, but `test_add_book` needs more work, but in a separate PR.  I don't understand why openlibrary/catalog/add_book/tests/test_add_book.py is running and failing. It should be excluded according to this line. It looks like Travis is not testing correctly against the contents of this PR. Any ideas?

Reply:
There are two test runs... scripts/run_doctests.sh and scripts/test_py3.sh.